images/manageiq-rpms/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM registry.access.redhat.com/ubi9/ubi
+
+COPY rpms/* /tmp/rpms/
+RUN  rm -f /tmp/rpms/.gitkeep
+
+RUN if [[ -n "$(ls /tmp/rpms)" ]]; then \
+      ARCH=$(uname -m) && \
+      dnf config-manager --setopt=tsflags=nodocs --setopt=install_weak_deps=False --save && \
+      dnf -y --setopt=protected_packages= remove redhat-release && \
+      dnf -y remove *subscription-manager* && \
+      dnf -y install \
+        http://mirror.stream.centos.org/9-stream/BaseOS/${ARCH}/os/Packages/centos-stream-release-9.0-24.el9.noarch.rpm \
+        http://mirror.stream.centos.org/9-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-9.0-24.el9.noarch.rpm \
+        http://mirror.stream.centos.org/9-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-9.0-24.el9.noarch.rpm ; \
+    fi
```

Comment:
Total aside, I'm tempted to move this into it's own image that is like ubi9-but-actually-centos9

Reply:
docker.io/manageiq-ubi9-😉😉:latest